api-tools/webhooks/testwebhooks.py

```diff
@@ -1,7 +1,34 @@
 from flask import Flask, request, make_response
+import os
 import json
 app = Flask(__name__)
 
+def verify_webhook(header, body, secret):
+    """Verifies that the webhook originated from Onfleet.
+    Args: 
+        header: value of the X-Onfleet-Signature header in raw bytes
+        body: should be the full body of the POST request in raw bytes, *not* the parsed JSON object
+        secret: the value of the webhook secret from the Onfleet dashboard, in hexadecimal format
+    Returns:
+        True for verified, False for not verified"""
+    import hashlib, hmac, binascii
+    return hmac.new(binascii.a2b_hex(secret), body, 'sha512').hexdigest() == header
+
+def warn_if_unverified(req):
+  if app.secret and not verify_webhook(req.headers['X-Onfleet-Signature'], req.data, app.secret):
+        print("Warning: could not verify the origin of the webhook invocation with provided secret key.")
+
+@app.before_first_request
+def check_for_secret():
+  if "WEBHOOK_SECRET" in os.environ:
+    app.secret = os.environ["WEBHOOK_SECRET"]
+  else:
+    print("\n***  NOTICE: Webhooks are running in unverified mode.  ***")
+    print("To verify webhooks, copy your Webhook secret key from the Onfleet dashboard "
+    "and set it in the environment as WEBHOOK_SECRET before running this script.\n\n"
+    "For example, export FLASK_APP=testwebhooks.py WEBHOOK_SECRET=XXXXXXXXXXXXXXXXXXXXX flask run")
```

Comment:
I would suggest adding this line in the README as well, specifically calling out the usage of the WEBHOOK_SECRET:

`export FLASK_APP=testwebhooks.py WEBHOOK_SECRET=XXXXXXXXXXXXXXXXXXXXX flask run`